modules/ossm-deploy-multi-mesh.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+//ALREADY COMMITTED IN DEPLOYMENT PR.  Update with capitalization changes.
+[id="ossm-deploy-multi-mesh_{context}"]
+= Multimesh or federated deployment model
+
+[role="_abstract"]
+*Federation* is a deployment model that lets you share services and workloads between separate meshes managed in distinct administrative domains.
+
+The Istio multi-cluster model requires a high level of trust between meshes, and requires remote access to all Kubernetes API servers on which the individual meshes reside. {ProductName} federation takes an opinionated approach to a multi-cluster implementation of Service Mesh that assumes _minimal_ trust between meshes.
+
+A *federated mesh* is a group of meshes behaving as a single mesh.  The services in each mesh may be unique services (that is, adding services to the mesh importing them), may provide additional workloads for the same services across the meshes (for example, providing high availability), or a combination of both.  All meshes that are joined into a federated mesh remain managed individually, and you must explicitly configure which services are exported to and imported from other meshes in the federation. Support functions such as certificate generation, metrics and trace collection remain local in their respective meshes.
```

Comment:
this is a really great paragraph! just the part in brackets after "services in each mesh may be unique services" I had to read twice-- maybe we can make that more explicit? potentially like "...unique services, which can be imported as new services into other meshes"? not sure